### ISSUE TITLE
[MAS1.4.1] [Visual Requirements-Debug] Visually “Show\Hide diff” button appearing as normal text 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1816]
**Actual Result:**
Visually “Show\Hide diff” button appearing as normal text present in the Debug screen 

**Expected Result:**
Visually “Show\Hide diff” control should appear as button 

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3. Bot screen upon connecting via azure/teams
4.Debug screen gets appeared on emulator 
5. Navigate to the “Show\Hide diff” button 
6. Observe the issue

**User Impact:**
If colour for button  is used as the only visual means of conveying information then users without perception of colour (e.g., colour blindness, without vision or limited vision) cannot observe, or see information conveyed through colour alone. So, chart’s content does not use only colour to convey meaning and provide another means by which the information is also communicated.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579}
![MAS1 4 1_Debug](https://user-images.githubusercontent.com/53518994/64164292-91371800-ce60-11e9-8647-ef38e46106d5.png)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue got fixed hence we are closing this bug.
